src/wallet/wallet_args.cpp

```diff
@@ -138,7 +139,7 @@ namespace wallet_args
       log_path = command_line::get_arg(vm, arg_log_file);
     else
       log_path = mlog_get_default_log_path("monero-wallet-cli.log");
```

Comment:
I don't understand this patch based on the title. Is this the offending line? Why would `wallet-rpc` log to console, but not `wallet-cli`. Or ?